aws-codeguruprofiler-profilinggroup/src/test/java/software/amazon/codeguruprofiler/profilinggroup/RequestBuilder.java

```diff
@@ -1,10 +1,14 @@
 package software.amazon.codeguruprofiler.profilinggroup;
 
+import java.util.Collections;
+
 import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
 
 public class RequestBuilder {
     static ResourceHandlerRequest<ResourceModel> makeValidRequest() {
-        return makeRequest(ResourceModel.builder().profilingGroupName("IronMan-Suit-34").build());
+        return makeRequest(ResourceModel.builder().anomalyDetectionNotificationConfiguration(Collections.singletonList(
+                Channel.builder().channelUri("arn:aws:sns:us-east-1:111111111111:SampleTopic").build())).profilingGroupName("IronMan" +
+                "-Suit-34").build());
     }
```

Comment:
Since we have dedicated tests for the anomaly behavior, I suggest undoing this change as well. (E.g. we also do not have here the optional compute platform).